Sources/Conformance/conformance.pb.swift

```diff
@@ -175,15 +175,15 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
       return nil
     }
 
-    fileprivate func traverse(visitor: SwiftProtobuf.Visitor, start: Int, end: Int) throws {
+    fileprivate func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V, start: Int, end: Int) throws {
```

Comment:
Notable change:  `traverse` is generic in the visitor type and accepts the visitor as an `inout` parameter.